superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -163,31 +176,59 @@ function ListViewCard({
             <GradientContainer>
               <CardCoverImg
                 src={imgURL}
-                onError={e => {
-                  e.currentTarget.src = imgFallbackURL;
-                }}
+                fallback={imgFallbackURL}
+                isLoading={loading}
               />
             </GradientContainer>
           </a>
           <CoverFooter className="cover-footer">
-            {coverLeft && <CoverFooterLeft>{coverLeft}</CoverFooterLeft>}
-            {coverRight && <CoverFooterRight>{coverRight}</CoverFooterRight>}
+            {!loading && coverLeft && (
+              <CoverFooterLeft>{coverLeft}</CoverFooterLeft>
+            )}
+            {!loading && coverRight && (
+              <CoverFooterRight>{coverRight}</CoverFooterRight>
+            )}
           </CoverFooter>
         </Cover>
       }
     >
-      <Card.Meta
-        title={
-          <>
-            <TitleContainer>
-              <TitleLink href={url}>{title}</TitleLink>
-              {titleRight && <div className="title-right"> {titleRight}</div>}
-              <div className="card-actions">{actions}</div>
-            </TitleContainer>
-          </>
-        }
-        description={description}
-      />
+      {loading && (
+        <Card.Meta
+          title={
+            <>
+              <TitleContainer>
+                <Skeleton.Input active style={{ width: 250 }} size="small" />
```

Comment:
NIT: class or constant. double diamonds `{{ ... }}` are generally suboptimal as they get re-created on each render and also trigger re-render in PureComponents (are they still popular?)

Reply:
You are correct, they were generally not favorable (though I'm not so sure anymore with the rise in popularity of functional components). However, these components just end up rendering divs with some loading shimmer styles, they're about a simple/dumb as components get. It won't have a measurable impact on performance in this instance. I'll move them to emotion though for consistency with other components.